components/global/portal/io.cellery.observability.ui/src/components/SignIn.js

```diff
@@ -60,9 +54,22 @@ const styles = (theme) => ({
     },
     submit: {
         marginTop: theme.spacing.unit * 3
+    },
+    centerDiv: {
+        position: "absolute",
+        margin: "auto",
+        top: 0,
+        right: 0,
+        bottom: 0,
+        left: 0,
+        width: "200px",
+        height: "100px"
+
     }
 });
 
+const idpAddress = "wso2-apim";
```

Comment:
Let's move this to constants.js file for now and add a "TODO" comment. (Need to move to portal config later)